go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -1177,3 +1190,21 @@ func (bnc *BaseNetworkController) wasPodReleasedBeforeStartup(uid, nad string) b
 	}
 	return bnc.releasedPodsBeforeStartup[nad].Has(uid)
 }
+
+func checkBridgeMapping(physicalNetworkName string) error {
+	external_ids, stderr, err := util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
```

Comment:
let's rename the variable; something like `ovsTableMappings` or something like that - you're grabbing a subset of the external ids, not all of them.